### ISSUE TITLE
Option to shift the center of the bar to the `zero` value

### ISSUE THREAD

**Issue by tcharlss** [IonDen/ion.rangeSlider#663]
Hi,

An image will explain it best, here's what I'm talking about :

![ionslider_negative_positive_range](https://user-images.githubusercontent.com/9882128/67315186-59fef200-f506-11e9-8dd8-947bdc983c07.png)

So, on a simple type, and when the values range from negative to positive, it would be nice to have an option to set the center of the bar on the `zero` value.

Currently I'm using the `onChange` callback to achieve this :
```javascript
function ajdustBar (data) {
	/* only for single type when '0' is in the values range */
	if (
		data.to_percent === 0
		&& ( 0 >= data.min && 0 <= data.max )
	) {
		var style = {};
		var $bar = $(data.slider[0]).find('.irs-bar');
		var total = Math.abs(data.min) + data.max;
		var center_percent = 100 * Math.abs(data.min) / total; // position du 0 en %
		if (data.from >= 0) {
			style.left = center_percent + '%';
			style.width = data.from_percent - center_percent + '%';
		} else {
			style.left = data.from_percent + '%';
			style.width = center_percent - data.from_percent + '%';
		}
		$bar.css(style);
	}
}
```

It works well, but unfortunately only after the first user interaction, is there no callback after the slider initialization?

Anyway, it would be nice if this feature was added as an option.

**Comment by tcharlss** [comment]
Also I have no idea how this parameter should be called.
`center_bar_on_zero` maybe ?

**Comment by IonDen** [comment]
Interesting Idea. Will think about implementation
